griptape/drivers/vector/qdrant_vector_store_driver.py

```diff
@@ -0,0 +1,212 @@
+    """
+
+    location: Optional[str] = field(default=None, kw_only=True, metadata={"serializable": True})
+    url: Optional[str] = field(default=None, kw_only=True, metadata={"serializable": True})
+    host: Optional[str] = field(default=None, kw_only=True, metadata={"serializable": True})
+    path: Optional[str] = field(default=None, kw_only=True, metadata={"serializable": True})
+    port: int = field(default=6333, kw_only=True, metadata={"serializable": True})
+    grpc_port: int = field(default=6334, kw_only=True, metadata={"serializable": True})
+    prefer_grpc: bool = field(default=False, kw_only=True, metadata={"serializable": True})
+    api_key: Optional[str] = field(default=None, kw_only=True, metadata={"serializable": True})
+    https: bool = field(default=None, kw_only=True, metadata={"serializable": True})
+    prefix: Optional[str] = field(default=None, kw_only=True, metadata={"serializable": True})
+    force_disable_check_same_thread: Optional[bool] = field(
+        default=False, kw_only=True, metadata={"serializable": True}
+    )
+    timeout: Optional[int] = field(default=5, kw_only=True, metadata={"serializable": True})
+    distance: str = field(default=DEFAULT_DISTANCE, kw_only=True, metadata={"serializable": True})
+    collection_name: str = field(kw_only=True, metadata={"serializable": True})
+    vector_name: Optional[str] = VECTOR_NAME
+    content_payload_key: str = field(default=CONTENT_PAYLOAD_KEY, kw_only=True, metadata={"serializable": True})
+
+    def __attrs_post_init__(self) -> None:
+        self.client = import_optional_dependency("qdrant_client").QdrantClient(
+            location=self.location,
+            url=self.url,
+            host=self.host,
+            path=self.path,
+            port=self.port,
+            prefer_grpc=self.prefer_grpc,
+            grpc_port=self.grpc_port,
+            api_key=self.api_key,
+            https=self.https,
+            prefix=self.prefix,
+            force_disable_check_same_thread=self.force_disable_check_same_thread,
+            timeout=self.timeout,
+        )
+
+    def delete_vector(self, vector_id: str) -> None:
+        """
+        Delete a vector from the Qdrant collection based on its ID.
+
+        Parameters:
+            vector_id (str): ID of the vector to delete.
+        """
+        # rest = import_optional_dependency("qdrant_client.http.models")
+        deletion_response = self.client.delete(
+            collection_name=self.collection_name,
+            points_selector=import_optional_dependency("qdrant_client.http.models").PointIdsList(points=[vector_id]),
+        )
+        if deletion_response.status == import_optional_dependency("qdrant_client.http.models").UpdateStatus.COMPLETED:
+            print(f"ID {vector_id} is successfully deleted")
+        else:
+            print(f"Failed to delete ID {vector_id}. Status: {deletion_response.status}")
```

Comment:
I recommend using logging instead of print statements.
Supports disabling if needed.